src/buildTree.js

```diff
@@ -12,13 +12,14 @@ const INHERITABLE_STYLES = [
   'fontSize',
   'fontStyle',
   'fontWeight',
+  'textAlign',
+  'textDecoration',
```

Comment:
Funny thing is that because of the expansion they both work `textDecoration` & `textDecorationLine`; Thoughts?